subprojects/ide/src/main/java/org/gradle/plugins/ide/api/PropertyListGeneratorTask.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.plugins.ide.api;
```

Comment:
Can you help me understand the use case for this external part of our API? I don't understand why a Gradle User would need to generate property list configuration files?  Shouldn't there be enough information in the gradle build file to figure out how to generate an XCode project and whatever properties files are needed to build a project using XCode?